### ISSUE TITLE
[Feature] Realtime Scraping 

### ISSUE THREAD

**Issue by Akilan1999** [HWTechClub/gitscraper#7]
To port the scraper with the repository [Custom-Website-Generator](https://github.com/HWTechClub/Custom-Static-Website-Generator).

We would need an Api call where we can provide a Github Username and get in response a JSON file

**Comment by FarazzShaikh** [comment]
Good Idea. We can provide a config in a request and have it scrape in real-time. But this will have to wait till the Core is complete.
